compiler/rustc_middle/src/ty/context.rs

```diff
@@ -1323,6 +1357,12 @@ impl<'tcx> TyCtxt<'tcx> {
         self.cstore.encode_metadata(self)
     }
 
+    // Note that this is *untracked* and should only be used within the query
+    // system if the result is otherwise tracked through queries
```

Comment:
```suggestion
    /// Note that this is *untracked* and should only be used within the query
    /// system if the result is otherwise tracked through queries
```